tasks/src/main/java/ru/sberbank/school/task11/Streams.java

```diff
@@ -0,0 +1,122 @@
+package ru.sberbank.school.task11;
+
+import java.util.*;
+import java.util.function.Function;
+import java.util.stream.Stream;
+
+/**
+ * Облегченная версия класса {@link Stream}
+ */
+public class Streams {
+
+    /**
+     * Принимает на вход массив элементов и возвращает стрим построенный на основе этих элементов.
+     *
+     * @param elements входные элементы
+     * @return стрим элементов из elements
+     */
+    public static Streams of(Object... elements) {
+        return null;
+    }
+
+    /**
+     * Принимает на вход коллекцию элементов и возвращает стрим построенный на основе этих элементов.
+     *
+     * @param elements входные элементы
+     * @return стрим элементов из elements
+     */
+    public static Streams of(Collection elements) {
+        return null;
+    }
+
+    /**
+     * Фильтрация элементов по заданному правилу. Необходимо подобрать нужый интерфейс для передачи в этот метод.
+     * <p>
+     * Иными словами, этот метод оставляет в коллекции только те элементы, которые удовлетворяют условию в лямбде.
+     * <p>
+     * Аналог Stream#map(Function).
+     * <p>
```

Comment:
&lt;p&gt; tag should be preceded with an empty line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a674809488>*